wiki/Tournaments/GBC/GBC_2024_Spring/en.md

```diff
@@ -0,0 +1,221 @@
+   - This rule should not to be abused. Referees may veto a rematch request if they find that this is the case.
+4. Unless there are special reasons, the organiser will only record each player's first score of the beatmap in the match room.
+5. The default rest time is **120 seconds**. If all players in the room agree to start early and get ready, the referee may start the match early.
+6. Players are expected to keep the match running fluently and without delays. Excessive delays from the players' side may result in penalties being applied at the referee's discretion. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
+7. All players and staff must be treated with respect. Instructions from referees and the host are to be followed. Decisions labelled as final are not to be objected.
+8. The multiplayer chat rooms underlie the [osu! community rules](/wiki/Rules).
+9. Disrupting the match by foul play or seed manipulation will result in disqualification, and any such players will be announced.
+10. Unexpected incidents will be handled by the host. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+11. The host may request liveplays or recordings of individual players at any point.
+12. The host reserves the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Mappool information
+
+1. Each group has four mappools, and each group uses one mappool per round.
+2. Each mappool consists of 7 beatmaps that will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
+3. The type of beatmap included in the mappool is not fixed.
+
+### Scheduling information
+
+1. Each stage of the tournament is played during **2 weeks**.
+   - Group EX will play in the same week as Group S, and Group A in the same week as Group B.
+2. All matches will be held between Saturday 08:00, and Sunday 23:59 (UTC+08), save for exceptional circumstances.
+3. The organiser will provide alternative tournament slots before the start of matches each week, creating a schedule that is satisfactory to all players.
+   - In case a special match time is required, please inform the organiser in advance. The organiser will try to fulfill the requirements of all players, but **cannot** make any guarantees.
+4. Matches may not be scheduled to any time **beyond Sunday, 23:59 (UTC+08)** of the week they are to be played at.
+   - Even if needed, the match must be completed by next Wednesday; later reschedule requests will be rejected.
+   - **Do not ask for a reschedule unless it is absolutely needed. The host reserves the right to deny any reschedule requests.**
+
+### Playoff stage
+
+The rules are largely the same as in the regular season.
```

Comment:
what does "season" mean here? it was never mentioned

Reply:
wrong typing, will change